src/main.rs

```diff
@@ -504,29 +497,29 @@ impl AsyncComponent for App {
             gtk::glib::Propagation::Stop
         });
 
-        #[cfg(target_os = "linux")]
-        let tray_img = Icon::from_png_bytes(TRAY_ICON).expect("Tray icon must exist");
+        #[cfg(any(target_os = "macos", target_os = "linux"))]
+        let tray_img = Icon::from_png_bytes(TRAY_ICON).expect("Tray icon exists and is a valid PNG; qe");
         #[cfg(target_os = "windows")]
-        let tray_img = Icon::from_resource(1, None).expect("Tray icon must exist");
+        let tray_img = Icon::from_resource(1, None).expect("Tray icon exists and is a valid PNG; qe");
 
         let tray = TrayIconBuilder::new()
             .with_icon(tray_img)
             .with_tooltip("Space Acres")
-            .with_menu(build_menu())
+            .with_menu(Menu::new([
+                MenuItem::button("Open", TrayMenuSignal::Open),
+                MenuItem::button("Quit", TrayMenuSignal::Quit),
+            ]))
             .build({
                 let sender = sender.clone();
                 move |tray_event| match tray_event {
                     TrayEvent::Menu(signal) => match signal {
-                        TrayMenuSignal::Open => sender.input(AppInput::ShouWindow),
+                        TrayMenuSignal::Open => sender.input(AppInput::ShowWindow),
                         TrayMenuSignal::Quit => sender.input(AppInput::Quit),
                     },
-                    TrayEvent::Tray(e) => match e {
-                        ClickType::Left => sender.input(AppInput::ShouWindow),
-                        _ => {}
-                    },
+                    _ => {}
                 }
             })
-            .expect("Must be able to build tray menu");
+            .expect("Icon is added in build script; qed");
```

Comment:
This has nothing to do with the build script though, so this proof is incorrect. What are the cases when this fails?